restapi/tenants.go

```diff
@@ -354,3 +368,52 @@ func getTenantCreatedResponse(token string, params admin_api.CreateTenantParams)
 
 	return err
 }
+
+// updateTenantAction does an update on the minioInstance by patching the desired changes
+func updateTenantAction(ctx context.Context, operatorClient OperatorClient, nameSpace string, params admin_api.UpdateTenantParams) error {
+	imageToUpdate := params.Body.Image
+	minInst, err := operatorClient.MinIOInstanceGet(ctx, nameSpace, params.Name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	// if image to update is empty we'll use the latest image by default
+	if strings.TrimSpace(imageToUpdate) != "" {
+		minInst.Spec.Image = params.Body.Image
+	} else {
+		minInst.Spec.Image = "minio/minio:latest"
```

Comment:
Instead of using the latest tag please use the function we have to retrieve the latest release

Reply:
`getLatestMinIOImage()`

Reply:
done @dvaldivia